src/gasman.c

```diff
@@ -1805,82 +1799,6 @@ UInt CollectBags (
 
     /* * * * * * * * * * * * * * * sweep phase * * * * * * * * * * * * * * */
 
-#if 0
-    /* call freeing function for all dead bags                             */
-    if ( NrTabFreeFuncBags ) {
```

Comment:
@stevelinton Do you by chance remember what this commented out code was good for?